### ISSUE TITLE
EYEPLUS_DEV from EYEPLUS sensors

### ISSUE THREAD

**Issue by Enjoy-Combi** [slydiman/ha_custom_onvif#6]
Hi,

Got this from the log:
b'<?xml version="1.0" encoding="UTF-8"?>\r\n<env:Envelope xmlns:env="http://www.w3.org/2003/05/soap-envelope" xmlns:soapenc="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:tst="http://www.onvif.org/ver10/storage/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:dn="http://www.onvif.org/ver10/network/wsdl" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:gte="http://www.onvif.org/ver10/gtextension/wsdl" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl" xmlns:wsoap12="http://schemas.xmlsoap.org/wsdl/soap12" xmlns:http="http://schemas.xmlsoap.org/wsdl/http" xmlns:d="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsadis="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:wsa="http://www.w3.org/2005/08/addressing" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrf-bf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:wsntw="http://docs.oasis-open.org/wsn/bw-2" xmlns:wsrf-rw="http://docs.oasis-open.org/wsrf/rw-2" xmlns:wsaw="http://www.w3.org/2006/05/addressing/wsdl" xmlns:wsrf-r="http://docs.oasis-open.org/wsrf/r-2" xmlns:tnsn="http://www.eventextension.com/2011/event/topics"><env:Body><tds:GetDeviceInformationResponse><tds:Manufacturer>EYEPLUS</tds:Manufacturer>\r\n<tds:Model>EYEPLUS_DEV</tds:Model>\r\n<tds:FirmwareVersion>5.6.1.1127</tds:FirmwareVersion>\r\n<tds:SerialNumber>12345679890</tds:SerialNumber>\r\n<tds:HardwareId>88</tds:HardwareId>\r\n</tds:GetDeviceInformationResponse>\r\n</env:Body>\r\n</env:Envelope>\r\n'

2021-12-28 09:10:45 DEBUG (MainThread) [custom_components.onvif] Camera Garage info = DeviceInfo(manufacturer='EYEPLUS', model='EYEPLUS_DEV', fw_version='5.6.1.1127', serial_number='12345679890', mac='41a0bc:41a0bd:41a0be:41a0bf:41a0c0:41a0c1')

Is this usefull ?

**Comment by slydiman** [comment]
Hi. 

It is hard to understand what is wrong. I don't see a error message.

I used the WireShark tool to record the full communication between HA and the camera to understand what is wrong.

For now I see only one serious problem in your message `mac='41a0bc:41a0bd:41a0be:41a0bf:41a0c0:41a0c1' `.
MAC address must contain only 6 bytes separated by colon, something like `mac='41:a0:bc:xx:xx:xx'`. Please open the new issue in [home-assistant/core](https://github.com/home-assistant) and provide this information (replace `custom_components.onvif` with `onvif`). Something is wrong in the MAC parser in HA Core.

**Comment by Enjoy-Combi** [comment]
Regarding the Mac address there is something strange. 
When I use the app onvier on my mobile, I get this strange address. 

Might it be an issue with the cam?
![Screenshot_20211228_184711_net biyee onvifer](https://user-images.githubusercontent.com/63100556/147595003-c57fd004-9246-491f-bded-6a19ee4fbe32.jpg)

**Comment by slydiman** [comment]
> Might it be an issue with the cam?

Right, something is wrong with the cam. Try to update the cam's firmware using the original app.

**Comment by Enjoy-Combi** [comment]
And my router gives 04:39:26:42:bb:a
Strange. 

Will report the issue to ha. 

Many thanks

**Comment by Enjoy-Combi** [comment]
The official app gives the right Mac address, nevertheless there is no firmware update available.

**Comment by Enjoy-Combi** [comment]
Hi,

FYI : https://github.com/home-assistant/core/issues/63399#issue-1093686255

And....happy new year :-) !
